testsuite/settings.py

```diff
@@ -15,7 +15,7 @@
 CONSUL_URL = os.environ.get('TESTING_CONSUL_URL', 'http://10.66.60.1:8500')
 NOMAD_URL = os.environ.get('TESTING_NOMAD_URL', 'http://10.66.60.1:4646')
 NOMAD_JOB_PREFIX = f'testsuite-{random_code(8)}-'
-NOMAD_DEPLOYMENT_NAME = f"test {datetime.now().strftime('%H:%M:%S')}"
+NOMAD_DEPLOYMENT_NAME = f"test-vmck@{os.environ.get('HOSTNAME', gethostname())}"  # noqa: E501
```

Comment:
The timestamp was intentional. It would be good to distinguish between to different test runs from the same host. And please extract a variable `hostname = os.environ.get('HOSTNAME', gethostname())` for clarity.